UnitsNet/GeneratedCode/Quantities/Force.g.cs

```diff
@@ -705,12 +708,30 @@ public static bool TryParseUnit(string str, IFormatProvider? provider, out Force
             return Acceleration.FromMetersPerSecondSquared(force.Newtons / mass.Kilograms);
         }
 
+        /// <summary>Get <see cref="Area"/> from <see cref="Force"/> / <see cref="Pressure"/>.</summary>
+        public static Area operator /(Force force, Pressure pressure)
+        {
+            return Area.FromSquareMeters(force.Newtons / pressure.Pascals);
+        }
+
         /// <summary>Get <see cref="Duration"/> from <see cref="Force"/> / <see cref="ForceChangeRate"/>.</summary>
         public static Duration operator /(Force force, ForceChangeRate forceChangeRate)
         {
             return Duration.FromSeconds(force.Newtons / forceChangeRate.NewtonsPerSecond);
         }
 
+        /// <summary>Get <see cref="ForceChangeRate"/> from <see cref="Force"/> / <see cref="Duration"/>.</summary>
+        public static ForceChangeRate operator /(Force force, Duration duration)
+        {
+            return ForceChangeRate.FromNewtonsPerSecond(force.Newtons / duration.Seconds);
+        }
+
+        /// <summary>Get <see cref="ForceChangeRate"/> from <see cref="Force"/> / <see cref="TimeSpan"/>.</summary>
+        public static ForceChangeRate operator /(Force force, TimeSpan timeSpan)
+        {
+            return ForceChangeRate.FromNewtonsPerSecond(force.Newtons / timeSpan.TotalSeconds);
+        }
```

Comment:
This may spawn a bigger discussion, but I'm thinking maybe we should clear up TimeSpan vs Duration a bit.

1. I like the idea of having `UnitsNet` using its own `Duration` as the primary type, and being consistent about it.
1. `Duration` already supports explicit cast to/from TimeSpan, let's change it to implicit cast.
2. Don't add duplicate method overloads in this PR just to support both types, implicit cast solves this.
3. I think we should also remove existing duplicate method overloads for TimeSpan/Duration, since we are doing a breaking change anyway.

I verified that with this change, then operator overloads also work with implicit cast of `TimeSpan`.

```cs
        [Fact]
        public void Foo()
        {
            Assert.Equal(ForceChangeRate.FromNewtonsPerSecond(100), Force.FromNewtons(100) / Duration.FromSeconds(1));
            Assert.Equal(ForceChangeRate.FromNewtonsPerSecond(100), Force.FromNewtons(100) / TimeSpan.FromSeconds(1));
        }
```

### Patch of `Duration` for implicit cast

```diff
diff --git forkSrcPrefix/UnitsNet/CustomCode/Quantities/Duration.extra.cs forkDstPrefix/UnitsNet/CustomCode/Quantities/Duration.extra.cs
index 4333d3c7450db4346bd224a8a0d152c075945e44..bac40311628a06a8700203e54f6557f166f8ee7e 100644
--- forkSrcPrefix/UnitsNet/CustomCode/Quantities/Duration.extra.cs
+++ forkDstPrefix/UnitsNet/CustomCode/Quantities/Duration.extra.cs
@@ -15,10 +15,7 @@ public partial struct Duration
         /// <returns>The TimeSpan with the same time as the duration</returns>
         public TimeSpan ToTimeSpan()
         {
-            if ( Seconds > TimeSpan.MaxValue.TotalSeconds ||
-                Seconds < TimeSpan.MinValue.TotalSeconds )
-                throw new ArgumentOutOfRangeException( nameof( Duration ), "The duration is too large or small to fit in a TimeSpan" );
-            return TimeSpan.FromTicks((long)(Seconds * TimeSpan.TicksPerSecond));
+            return TimeSpan.FromSeconds(Seconds);
         }
 
         /// <summary>Get <see cref="DateTime"/> from <see cref="DateTime"/> plus <see cref="Duration"/>.</summary>
@@ -33,14 +30,14 @@ public TimeSpan ToTimeSpan()
             return time.AddSeconds(-duration.Seconds);
         }
 
-        /// <summary>Explicitly cast <see cref="Duration"/> to <see cref="TimeSpan"/>.</summary>
-        public static explicit operator TimeSpan(Duration duration)
+        /// <summary>Implicitly cast <see cref="Duration"/> to <see cref="TimeSpan"/>.</summary>
+        public static implicit operator TimeSpan(Duration duration)
         {
             return duration.ToTimeSpan();
         }
 
-        /// <summary>Explicitly cast <see cref="TimeSpan"/> to <see cref="Duration"/>.</summary>
-        public static explicit operator Duration(TimeSpan duration)
+        /// <summary>Implicitly cast <see cref="TimeSpan"/> to <see cref="Duration"/>.</summary>
+        public static implicit operator Duration(TimeSpan duration)
         {
             return FromSeconds(duration.TotalSeconds);
         }
```